database/mongoDB/redis.js

```diff
@@ -0,0 +1,30 @@
+const { loadProject } = require('./db.js');
+
+var redisClient = require('redis').createClient;
+
+// const db = require('./db.js');
+var hits = 0;
+var total = 0;
+
+const redis = redisClient(6379, 'localhost');
+const cacheGetProject =  function(id, cb){

```

Comment:
I think that the readability of this function (and, upstream, the get handler) would be really helped by promisifying it (with e.g. Bluebird), and possibly using async/await.

Reply:
I agree.  but this will get pushed to tech debt as I want to get the function up and working before form,